elastic/apm-server
# Deprecate `apm-server apikey`

## Motivation/summary

Deprecate the `apikey` commands, as users have better options now. See https://github.com/elastic/apm-server/issues/9444.

## Checklist

~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~

## How to test these changes

1. Run `apm-server -h`, verify the `apikey` command mentions that it is deprecated.
2. Run `apm-server apikey -h`, verify the command description mentions that it is deprecated.
3. Run `apm-server apikey create`, verify the command output mentions that it is deprecated.

## Related issues

Closes https://github.com/elastic/apm-server/issues/9444
